recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: cmocean-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/c/cmocean/cmocean-{{ version }}.tar.gz
```

Comment:
> huh, they are lowercase on both my and matplotlib's github version. I will investigate...

Yep. I can confirm that the GitHub tarball is OK. We can use it here instead of PyPI (and you can do a post release there to avoid issuing a new release.)

Just change it here to:

```
url: https://github.com/matplotlib/cmocean/archive/{{ version }}.tar.gz
sha256: 22e40329f5fd1c6d2136d7b32dfe988f3b8570a360b86bf9786efb2cb20c9aaa
```